src/cdk-experimental/listbox/listbox.ts

```diff
@@ -297,6 +303,12 @@ export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
     } else {
       manager.onKeydown(event);
     }
+
+    /** Will select an option if shift was pressed while navigating to the option */
+    const isArrow = (keyCode === UP_ARROW || keyCode === DOWN_ARROW);
+    if (isArrow && event.shiftKey && previousActiveIndex !== this._listKeyManager.activeItemIndex) {
```

Comment:
What's the thinking behind toggling options based on arrow keys with shift rather than requiring a separate space/enter keypress? Was this from the aria authoring practices?

Reply:
So it would work with a separate space/enter keypress. In the authoring practices it says that selection via SHIFT + arrow key is an optional way to do selection.